src/pykeen/nn/node_piece.py

```diff
@@ -199,15 +221,15 @@ def extra_repr(self) -> Iterable[str]:  # noqa: D102
         yield f"alpha={self.alpha}"
         yield f"epsilon={self.epsilon}"
 
-    def __call__(self, edge_index: numpy.ndarray) -> numpy.ndarray:  # noqa: D102
+    def __call__(self, edge_index: numpy.ndarray, known_anchors: numpy.ndarray) -> numpy.ndarray:  # noqa: D102
         # convert to sparse matrix
         adj = _edge_index_to_sparse_matrix(edge_index=edge_index)
         # symmetrize
         # TODO: should we add self-links
         # adj = (adj + adj.transpose() + scipy.sparse.eye(m=adj.shape[0], format="coo")).tocsr()
         adj = (adj + adj.transpose()).tocsr()
         # degree
-        degree_inv = numpy.reciprocal(numpy.asarray(adj.sum(axis=0)))[0]
+        degree_inv = numpy.reciprocal(numpy.asarray(adj.sum(axis=0), dtype=numpy.float))[0]
```

Comment:
That was quite a big bug - default dtype was int and `reciprocal` returned all-zero vector, hence PR was computed incorrectly converging to `self.alpha` - I checked in the debugger that all the values were exactly the same. 

Changing dtype to `numpy.float` fixed the problem

Reply:
Using `numpy.float` directly raises a `DeprecationWarning` from 1.20 onwards, cf. https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated

the new preferred way is to directly use `float` as dtype.

Reply:
done https://github.com/pykeen/pykeen/pull/776/commits/7672aecb7c6710da467364aad440dc7955b892e7